ui/src/labels/actions/index.ts

```diff
@@ -104,24 +118,38 @@ export const createLabel = (
   } = getState()
 
   try {
-    const createdLabel = await client.labels.create({
-      orgID: org.id,
-      name,
-      properties: properties as ILabelProperties,
+    const resp = await apiPostLabel({
+      data: {
+        orgID: org.id,
+        name,
+        properties: properties as ILabelProperties,
+      },
     })
 
+    if (resp.status !== 201) {
+      throw new Error(resp.data.message)
+    }
+
+    const createdLabel = addLabelDefaults(resp.data.label as Label)
```

Comment:
the typing on addLabelDefaults should allow you to remove all `as`'es in this file